ng2-bootstrap.ts

```diff
@@ -16,3 +31,24 @@ export * from  './components/typeahead';
 export * from  './components/position'
 export * from  './components/common'
 export * from  './components/ng2-bootstrap-config';
+
+export default {
+  directives: [Dropdown, Progress, Tab, TabHeading],
+  components: [
+    Accordion,
+    AccordionPanel,
```

Comment:
For consistency, we should keep everything in `directives` since the documentation states that. I can make a PR for this change if you'd like.